articles/azure/azs-how-create-vm-image-portal.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How to create a custom image from a VM on Azure Stack - Portal | UKCloud Ltd
+description: Provides details on how to create a custom image from a VM using the portal
+services: 
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM - Portal
+toc_fullpath: Users/How To/azs-how-create-vm-image-portal.md
+toc_mdlink: azs-how-create-vm-image-portal.md
+---
+
+# How to create a custom image from a VM on Azure Stack - Portal
```

Comment:
How to create a custom image from a VM on Azure Stack using the UKCloud Azure Stack portal